screensaver.bing/resources/skins/default/skin.xml

```diff
@@ -0,0 +1,10 @@
+<skin>
```

Comment:
Not really sure one of the legacy things... i've always left alone. I'll remove it.